jobs/JREGIONAL_PROCESS_BUFR

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# This J-JOB script runs the preprocess of BUFR files
+#      for the FV3-LAM model
```

Comment:
It would be helpful to include what type of process this does, and which executables are called. Perhaps @hu5970 could shed some light on exactly what the processing is?

Reply:
I have added the description from the associated issue for this PR.